templates/index.html

```diff
@@ -2,40 +2,126 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
-  <div class="jumbotron">
+  <div class="jumbotron" style="border-style: solid; border-color: darkgray; border-width: 1px;">
     <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
-
-  <div class="row">
-    <div class="col-md-12">
-      <h2>Items</h2>
-      <div class="list-group">
-        {% for item in items %}
-        <a href="#" class="list-group-item list-group-item-action">
-          <div class="d-flex w-100 justify-content-between">
-            <h5 class="mb-1">{{ item.position }}: {{ item.name }}</h5>
-            <small>status: {{ item.status }}</small>
-            <small>description: {{ item.description }}</small>
-            {% if item.status == "To Do" %}
-            <form action="complete/{{ item.id }}" method="post">
-              <button class="btn btn-outline-primary" type="submit">Complete</button>
+  <h1>Items</h1>
+  <table class="table table-dark table-hover">
+    <thead class="thead-light">
+      <tr>
+        <th scope="col" style="font-size: 22px;">TODO</th>
+        <th scope="col">Item</th>
+        <th scope="col">Status</th>
+        <th scope="col">Description</th>
+        <th scope="col">Action</th>
+      </tr>
+    </thead>
+    <tbody id="todo-tbody">
+      {% for item in view_model.todo_items %}
+        <tr>
+          <td></td>
+          <td>{{ item.name }}</td>
+          <td>{{ item.status }}</td>
+          <td>{{ item.description }}</td>
+          <td>
+            <form action="start/{{ item.id }}" method="post">
+              <button class="btn btn-outline-primary" type="submit" id="start-btn">Start</button>
```

Comment:
This shouldn't really have an `id` of start-btn. "id" should be unique out of all the html elements on the page, but you have one of these for each todo item.

You should make it a class or a name